### PR TITLE
PIMS-1250 Update healthchecksui to use appsettings.json

### PR DESCRIPTION
**Pull request by devinleighsmith**
add new secret for webhook which overwrites appsettings uri.
revert -sso related changes.

### CODE REVIEW

**Comment by devinleighsmith** [comment]
I'm happy with how this is working locally now - should work in openshift as well based on my preliminary testing.

File: backend/README.md [hunk 1]
```diff
@@ -12,3 +12,9 @@ The API is configured to run in a Docker container and has the following depende
 - [DAL](./docs/DAL.md) - Data Access Layer information.
 - [Tools](./docs/TOOLS.md) - Tooling information.
 - [Versioning](./docs/VERSIONING.md) - Version information.
+
+## Health Checks UI
+
+Health checks are configured to run against /api/live and /api/ready endpoints. These correspond to the kubernetes concepts of liveliness and readiness, see [here](https://docs.openshift.com/container-platform/3.11/dev_guide/application_health.html).
```
Comment (Fosol) [inline backend/README.md]: It isn't critical lol, but I create a `HealthController`, perhaps the endpoints should be `/api/health/live` and `/api/health/ready` instead?

File: backend/api/Startup.cs [hunk 2]
```diff
@@ -146,15 +146,7 @@ public void ConfigureServices(IServiceCollection services)
...
-            services.AddHealthChecksUI(setupSettings: setup =>
-            {
-                setup.AddHealthCheckEndpoint("liveliness", "http://localhost/live");
-                setup.AddHealthCheckEndpoint("readiness", "http://localhost/ready");
-                setup.AddWebhookNotification("rocketchat",
-                    uri: Configuration["ROCKETCHAT_HEALTH_HOOK"],
-                    payload: Configuration["ROCKETCHAT_HEALTH_PAYLOAD"],
-                    restorePayload: Configuration["ROCKETCHAT_HEALTH_RESTORE"]);
-            });
+            services.AddHealthChecksUI();
```
Comment (Fosol) [inline backend/api/Startup.cs]: Love how it pulls it all in with the config below!

File: backend/api/appsettings.json [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+      {
+        "Name": "readiness",
+        "Uri": "http://localhost/api/ready"
+      }
+    ],
+    "Webhooks": [
+      {
+        "Name": "RocketChat",
+        "Uri": "overwrite with env vars",
+        "Payload": "{\"text\":\"The HealthCheck [[LIVENESS]] is failing with the error message [[FAILURE]]. [[DESCRIPTIONS]].\"}",
```
Comment (Fosol) [inline backend/api/appsettings.json]: Can you point or document the keyword stuff somewhere?

File: openshift/templates/secrets/README.md [hunk 4]
```diff
@@ -50,22 +50,40 @@ oc process -f database-secrets.yaml -p ENV_NAME=prod | oc create -f -
 cd openshift/templates/app/secrets
 
 oc project $namespace_dev
-oc process -f api-secrets.yaml \
+oc process -f api-sso-secrets.yaml \
```
Comment (Fosol) [inline openshift/templates/secrets/README.md]: Good idea to keep our secrets in separate collections.

Comment (asanchezr) [reply openshift/templates/secrets/README.md]: Agree

File: openshift/templates/secrets/README.md [hunk 5]
```diff
@@ -50,22 +50,40 @@ oc process -f database-secrets.yaml -p ENV_NAME=prod | oc create -f -
...
   -p ENV_NAME=test \
   -p KEYCLOAK_AUTHORITY=[sso-test realm URL] \
   | oc create -f -
 
+oc project $namespace_test
+oc process -f api-health-secrets.yaml \
+  -p ENV_NAME=test \
+  -p KEYCLOAK_AUTHORITY=[health-test realm URL] \
+  | oc create -f -
+
```
Comment (asanchezr) [inline openshift/templates/secrets/README.md]: I believe the new health-secrets don't have a "KEYCLOAK_AUTHORITY" parameter... did you mean "ROCKETCHAT_HEALTH_HOOK_URI" by any chance?

**Pull request merged**